src/agent/stacktrace-parser/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "stacktrace-parser"
+version = "0.1.0"
+authors = ["Brian Caswell <bmc@shmoo.com>"]
```

```suggestion
authors = ["fuzzing@microsoft.com"]
```
(and grep for any other occurrences)